easybuild/easyconfigs/m/MATLAB/README.md

```diff
@@ -0,0 +1,32 @@
+This example uses Matlab 2018a and MATLAB-2018a.eb  
+Steps to install Matlab:
+1. Install the appropriate Java install which requires manual configuration. See Java documentation for install.
```

Comment:
I'd suggest changing "configuration" to "manual download of the source" (I was looking for a configuration option, and didn't find any). What about including a link to the Java documentation?

Reply:
Yes it is. But you could also ignore it and let MATLAB use its internal Java tree. This is what we are doing now.